_posts/2021-06-23-cockpit-247.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 247
+author: kkoukiou
+date: 2021-06-23
+tags: cockpit linux
+slug: cockpit-247
+category: release
+summary: Cockpit with Metrics and Virtual Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 247 and Cockpit Machines version 246.
+
+## Metrics: Grafana client setup
+
+The "Metrics settings" button introduced in [Cockpit 245](https://cockpit-project.org/blog/cockpit-245.html)
+has a new switch to set up exporting metrics to the network through the [PCP pmproxy service](https://linux.die.net/man/1/pmproxy)
+and the [Redis database](https://grafana-pcp.readthedocs.io/en/latest/datasources/redis.html):
+
+![metrics settings](/images/metrics-settings-2.png)
+
+If the machine uses firewalld, you can allow pmproxy in a particular zone afterwards:
+
+![metrics firewall zone selector](/images/metrics-fw-sel.png)
+
+After that, a remote [Grafana](https://grafana.com/) instance can read the machine's metrics through the
+[PCP Grafana plugin](https://grafana-pcp.readthedocs.io/), as described on the [PCP metrics feature page](https://cockpit-project.org/guide/latest/feature-pcp.html).
+
+## Machines: Share host files with the guest
```

Comment:
I wonder if it's worth mentioning "virtio-fs" directly in the title.  The people interested in this feature may already know what it's called?

Reply:
Something like this?

```suggestion
## Machines: Share host files with the guest using virtio-fs
```

Reply:
So they just got back to me again:

kkoukiou: groug dgilbert Lowercase "virtiofs" is the most recent spelling. Before it was virtio-fs but the Linux file system name used on the mount command-line is "virtiofs", so we recently agreed to try to use that consistently.
It doesn't help that we changed the spelling :-)
But the hope is that people won't try mount -t virtio-fs and receive an error :)

So I will adjust accordingly.